src/components/common/Footer/index.tsx

```diff
@@ -37,38 +38,45 @@ const Footer = (): ReactElement | null => {
   return (
     <footer className={css.container}>
       <ul>
-        <li>
-          <Typography variant="caption">&copy;2022–{new Date().getFullYear()} Core Contributors GmbH</Typography>
-        </li>
-        <li>
-          <Link href={AppRoutes.terms} passHref>
-            <MUILink>Terms</MUILink>
-          </Link>
-        </li>
-        <li>
-          <Link href={AppRoutes.privacy} passHref>
-            <MUILink>Privacy</MUILink>
-          </Link>
-        </li>
-        <li>
-          <Link href={AppRoutes.licenses} passHref>
-            <MUILink>Licenses</MUILink>
-          </Link>
-        </li>
-        <li>
-          <Link href={AppRoutes.imprint} passHref>
-            <MUILink>Imprint</MUILink>
-          </Link>
-        </li>
-        <li>
-          <Link href={AppRoutes.cookie} passHref>
-            <MUILink>Cookie Policy</MUILink>
-          </Link>
-          &nbsp;&mdash;&nbsp;
-          <Link href={`${router.pathname}#`} passHref>
-            <MUILink onClick={onCookieClick}>Preferences</MUILink>
-          </Link>
-        </li>
+        {IS_OFFICIAL_HOST ? (
+          <>
+            <li>
+              <Typography variant="caption">&copy;2022–{new Date().getFullYear()} Core Contributors GmbH</Typography>
```

Comment:
We could put this into an env var as well, then forks won't have to modify this code.